carbonmark-api/package.json

```diff
@@ -31,6 +31,7 @@
     "@sinclair/typebox": "^0.28.5",
     "dotenv": "^16.1.4",
     "ethers": "^5.7.2",
+    "ethers-v6": "npm:ethers@^6.7.1",
```

Comment:
Is this to avoid having to uplift the rest of the api that uses v5?

Reply:
Yes just getting a head start on the migration, Makka will get it done after the v2 work settles down